vv_trend/vv_trend.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+import shutil
+from pathlib import Path
+import numpy as np
+import matplotlib
+
+if __name__ == "__main__":
```

Comment:
I think that https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.switch_backend.html can be used to avoid this familiar old pattern. So basically the same idea but do the switch after the imports.

Reply:
It seems like with modern matplotlib we don't care and the new practice is just to use matplotlib.use('Agg') to specify this in main (I'm looking at fss_check for example) instead of needing to use switch_backend.  Do you agree?

Reply:
I did a little testing and it looks like this pattern should be reliable in scripts like this.